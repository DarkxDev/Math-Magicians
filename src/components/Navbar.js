import { Link } from 'react-router-dom';

const Navbar = () => (
  <div className="header">
    <h1>Math Magicians</h1>
    <ul className="nav-menu">
      <li><Link to="/">Home</Link></li>
      <li><Link to="/calculator">Calculator</Link></li>
      <li><Link to="/quote">Quote</Link></li>
    </ul>
  </div>

);

export default Navbar;
