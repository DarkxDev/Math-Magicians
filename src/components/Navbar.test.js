import React from 'react';
import { MemoryRouter } from 'react-router';
import renderer from 'react-test-renderer';
import Navbar from './Navbar';

test('Renders correctly', () => {
  const tree = renderer.create(
    <MemoryRouter>
      <Navbar />
    </MemoryRouter>,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
