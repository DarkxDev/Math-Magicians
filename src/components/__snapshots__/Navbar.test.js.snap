// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<div
  className="header"
>
  <h1>
    Math Magicians
  </h1>
  <ul
    className="nav-menu"
  >
    <li>
      <a
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
    </li>
    <li>
      <a
        href="/calculator"
        onClick={[Function]}
      >
        Calculator
      </a>
    </li>
    <li>
      <a
        href="/quote"
        onClick={[Function]}
      >
        Quote
      </a>
    </li>
  </ul>
</div>
`;
